.coveragerc

```diff
@@ -544,6 +544,7 @@ omit =
     homeassistant/components/sensor/supervisord.py
     homeassistant/components/sensor/swiss_hydrological_data.py
     homeassistant/components/sensor/swiss_public_transport.py
+    homeassistant/components/sensor/switcher_kis.py
```

Comment:
Delete

Reply:
Thank you.
Fixed in [this](https://github.com/home-assistant/home-assistant/pull/20830/commits/6d478ac3f47207b344a670fc3a04f1811c544a83) commit.